src/main/groovy/org/pih/warehouse/requisition/RequisitionStatus.groovy

```diff
@@ -80,14 +80,19 @@ enum RequisitionStatus {
         [CREATED, EDITING, APPROVED, PICKING, PICKED, CHECKING, ISSUED, CANCELED, PENDING, REQUESTED, DISPATCHED]
     }
 
-    // Options for request list when current location is supporting request approval (Added approved and waiting for approval)
+    // Options for request list when current location is supporting request approval (Added approved, rejected and waiting for approval)
```

Comment:
Can we add some additional context for each of these to provide a reason why we are restricting the options? You could potentially find this information in the ticket.